src/Object/Api/Mastodon/Status.php

```diff
@@ -115,9 +115,9 @@ public function __construct(array $item, Account $account, stdClass $count)
 		$this->language = null;
 		$this->uri = $item['uri'];
 		$this->url = $item['plink'] ?? null;
-		$this->replies_count = $count->replies;
-		$this->reblogs_count = $count->reblogs;
-		$this->favourites_count = $count->favourites;
+		$this->replies_count = $count->__get('replies');
+		$this->reblogs_count = $count->__get('reblogs');
+		$this->favourites_count = $count->__get('favourites');
```

Comment:
The magic method `__get` allows us to directly refer to missing/non-public properties. The property name will be passed to the `__get()` method. There is a similar `__set($name, $value)` method triggered when trying to set a non-existant/non-public property.
```suggestion
		$this->replies_count = $counts->replies;
		$this->reblogs_count = $counts->reblogs;
		$this->favourites_count = $counts->favourites;
```

Reply:
So `_get` and `__set` act like invisible getter/setter routines?